File: POO - Python & Java/practica-1/src/gestorAplicacion/Cita.java
package gestorAplicacion;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.Date;

public class Cita  implements Serializable{
	
	/*	CLASS VARIABLES	*/
	private static final long serialVersionUID = 1L;
	public static ArrayList<Integer> vetados;
	
	/*	INSTANCE VARIABLES	*/
	private Date fecha;
	private Paciente paciente;
	private Doctor doctor;
	private Quirofano quirofano;
	
	/*	CONSTRUCTORS	*/
	public Cita(Date fecha, Paciente paciente, Doctor doctor) {
		this(fecha, paciente, doctor, null);
	}
	
	public Cita(Date fecha, Paciente paciente, Doctor doctor, Quirofano quirofano) {
		this.fecha = fecha;
		this.paciente = paciente;
		this.doctor = doctor;
		this.quirofano = quirofano;
	}
	
	
	/*	GETTERS & SETTERS	*/
	public Date getFecha() { return this.fecha; }
	public void setFecha(Date fecha) { this.fecha = fecha; }
	
	public Paciente getPaciente() { return this.paciente; }
	public void setPaciente(Paciente paciente) { this.paciente = paciente; }
	
	public Doctor getDoctor() { return this.doctor; }
	public void setDoctor(Doctor doctor) { this.doctor = doctor; }
	
	public Quirofano getQuirofano() { return this.quirofano; }
	public void setQuirofano(Quirofano quirofano) { this.quirofano = quirofano; }
}

File: POO - Python & Java/practica-2/gestorAplicacion/Cita.py
class Cita:

    vetados=[]
    def __init__(self, fecha, paciente, doctor, quirofano):

        self._fecha = fecha
        self._paciente = paciente
        self._doctor = doctor
        self._quirofano = quirofano


    #	GETTERS & SETTERS	
    def getFecha(self):
        return self._fecha
    def setFecha(self, fecha):
        self._fecha = fecha

    def getPaciente(self):
        return self._paciente
    def setPaciente(self, paciente):
        self._paciente = paciente

    def getDoctor(self):
        return self._doctor
    def setDoctor(self, doctor):
        self._doctor = doctor

    def getQuirofano(self):
        return self._quirofano
    def setQuirofano(self, quirofano):
        self._quirofano = quirofano
